buffer/src/main/java/io/netty5/buffer/api/adaptor/ByteBufBuffer.java

```diff
@@ -452,7 +453,7 @@ public Buffer split(int splitOffset) {
         int woff = writerOffset();
         int roff = readerOffset();
         var drop = unsafeGetDrop().fork();
-        var splitBuffer = new ByteBufBuffer(control, delegate.slice(0, splitOffset), drop);
+        var splitBuffer = new ByteBufBuffer(control, delegate.retainedSlice(0, splitOffset), drop);
```

Comment:
@chrisvest this is the one issue you talked about ?

Reply:
Yes